data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			choice
+				`	(Say nothing.)`
+					goto explain
+				`	"You could have told me before."`
+			`	"Yes, but where is the fun in that? We also wanted to see how you would react.`
+			label explain
+			`	"We have a very interesting proposition for you: should you agree to bring supplies to our brethren, the Unfettered Hai, we will sell this ship to you for the modest price of 6.5 million credits. This is a very useful ship, and you should be able to repay us for this favor with ease by running more of the same kind of jobs. These supplies are needed on short notice, on <destination>, and we lack the time and personnel to bring them."`
+			`	"Of course, the ship has some... drawbacks, compared to its golden time, and you should not expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! It can also be of great use for mining, with all of the inbuilt equipement for it and its large cargo capacity. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I have no interest in this deal."`
+			`	"A shame. We will probably not meet again," they say, before signaling the ship to submerge into the ocean again.`
+				goto trainback
+			label take
+			action
+				give ship "Ancient Ladybug" "Good Omen"
+				fine 6500000
+				log "Accepted a somewhat shady deal for an old Hai ship, a Ladybug, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you have received notice of the fine being issued before they have even finished speaking. "We will need to make some modifications to the anti-missile system of the ship before you can use it. I also strongly would advise against repainting it, you will understand why later. I would not bother removing the rust, either, if I were you, it is not worth the risk."`
+			`	"Oh, and I almost forgot: make sure you return to the spaceport to get the cargo loaded onto your ship! And use it as your flagship, naturally."`
+				accept
+			label foolish
+			`	You get your weapon out and proceed to shoot on the ship.`
+			branch miss
+				random < 30
+			branch lucky?
+				random < 2
+			branch lucky??
+				random < 40
+			`	The shot reaches the ship, but the small weapon has no chance of doing anything to the heavy armor, even in its current state. The ship's speakers then activate, almost deafening you. "MY TURN!"`
```

Comment:
it was told to be quiet but in this case it doesnt listen because it knows you're not gonna be accepted anyway

Reply:
Sure, but doesn't change the fact that, to the reader, this is still weird. I'd still recommend removing it or providing the context you just explained in the mission itself.

Reply:
That's helpful, yes. Upon another viewing, this still feels very [Chekov's gun-ish](https://en.wikipedia.org/wiki/Chekhov%27s_gun). Basically, I think the fact that the AI has a voice it can use should be utilized later or removed entirely. Could it maybe comment in the missions where it returns to the Unfettered, or perhaps a unique comment if it's destroyed?

Reply:
gotcha, it commenting when it exits can be fun